sycl-jit/jit-compiler/CMakeLists.txt

```diff
@@ -40,6 +49,8 @@ target_include_directories(sycl-jit
   SYSTEM PRIVATE
   ${LLVM_MAIN_INCLUDE_DIR}
   ${LLVM_SPIRV_INCLUDE_DIRS}
+  ${CMAKE_SOURCE_DIR}/../clang/include
```

Comment:
Does LLVM CMake not define a variable for that?

Reply:
Changed this to use `${LLVM_EXTERNAL_CLANG_SOURCE_DIR}`.

Unfortunately there doesn't seem to be an equivalent for the build directory, from which an `.inc` file that defines `CLANG_VERSION_MAJOR` is included.